libbeat/dashboards/import_dashboards.go

```diff
@@ -0,0 +1,559 @@
+package main
+
+import (
+	"archive/zip"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/common/fmtstr"
+	"github.com/elastic/beats/libbeat/outputs/elasticsearch"
+	"github.com/elastic/beats/libbeat/outputs/outil"
+)
+
+type Options struct {
+	KibanaIndex string
+	ES          string
+	Index       string
+	Dir         string
+	File        string
+	Beat        string
+	Url         string
+	User        string
+	Pass        string
+}
+
+type CommandLine struct {
+	flagSet *flag.FlagSet
+	opt     Options
+}
+
+type Importer struct {
+	cl     *CommandLine
+	client *elasticsearch.Client
+}
+
+func ParseCommandLine() (*CommandLine, error) {
+	var cl CommandLine
+
+	cl.flagSet = flag.NewFlagSet("import", flag.ContinueOnError)
+
+	cl.flagSet.Usage = func() {
+		const usage = `
```

Comment:
Could you move this doc string somewhere on the top and just use the variable here? This would make the code more readable.